doc/style.rst

```diff
@@ -72,8 +82,9 @@ Tools
 
 * Autoformatter: `clang-format <https://clang.llvm.org/docs/ClangFormat.html>`_.
 
-  * Run: ``./run-clang-format.py -r .`` to see needed changes.
-  * Run: ``clang-format -i file.c`` to apply the changes.
+  * Run: ``pre-commit run --all-files --hook-stage manual`` to apply changes to
+    the whole repository. You must have a ``conda`` installation in your
+    ``PATH`` the first time you run this on a system.
```

Comment:
The first time?
```suggestion
    ``PATH`` to run this.
```

Reply:
After the first time, `pre-commit` will use the environment from the cache. As you suggest, we don't necessarily need to call that out.